src/libAtomVM/opcodesswitch.h

```diff
@@ -1116,10 +1116,12 @@ static void destroy_extended_registers(Context *ctx, unsigned int live)
         } else {                                                        \
             fun_module = globalcontext_get_module(ctx->global, module_name); \
             if (IS_NULL_PTR(fun_module)) {                              \
+                SET_ERROR(UNDEF_ATOM);                                  \
```

Comment:
What was happening before of this change? Did we leave anything uninitialized?

Reply:
Before this change, the process would crash based on garbage x[1] and x[2].

Reply:
Ok, thanks, so it is a bug also for release-0.6, lets's rebase it.

Reply:
I also added a line in CHANGELOG.md